src/core/agent/services/identifierService.ts

```diff
@@ -378,6 +378,12 @@ class IdentifierService extends AgentService {
         return { state: aid.response };
       })
     );
+    const notLinkedContacts = otherIdentifierContacts.filter(
+      (contact) => contact.groupId !== ourMetadata.groupMetadata?.groupId
+    );
+    if (notLinkedContacts.length) {
+      // Todo: I got confused about this part, I will update it later
+    }
```

Comment:
We just want to check that each of the `otherIdentifierContacts` contains the correct `groupId`. We don't need to check other contacts

Reply:
`When creating an OOBI URL for a member identifier, append the following query param to the end of the URL ?groupid=<uuid>`

`When scanning a KERI OOBI, if it contains the groupid query parameter and the UUID is recognised from one of our member identifiers, that contact should be linked to the member identifier`

`Adjust the create multi-sig interface to only allow creation of a multi-sig inception event using KERI contacts that are linked to the member identifier
`
https://cardanofoundation.atlassian.net/browse/DTIS-718
@iFergal  so when we create an oobi URL, we append our member identifier groupId (“uuid A”) to it. Then when another users scans it, how can this “uuid A” equals to their uuid?

Reply:
```
const notLinkedContacts = otherIdentifierContacts.filter(
      (contact) => contact.groupId !== ourMetadata.groupMetadata?.groupId
    );
```
This is how I find contacts that don't contain the correct groupId. If this array's length is bigger than 0, we throw the error.

Reply:
I replied on the ticket

Reply:
> This is how I find contacts that don't contain the correct groupId. If this array's length is bigger than 0, we throw the error.

My mistake yes, sorry thought you fetched all contacts. Makes sense